README.md

```diff
@@ -46,6 +46,9 @@ When reading files the API accepts several options:
   * `FAILFAST`: aborts with a RuntimeException if encounters any malformed line
 * `charset`: defaults to 'UTF-8' but can be set to other valid charset names
 * `inferSchema`: automatically infers column types. It requires one extra pass over the data and is false by default
+* `comment`: skip lines beginning with this character. Note the default behaviours are different for each parser:
```

Comment:
I discussed it with @rxin and @JoshRosen. In this case, it seems keeping a uniform behavior is a better choice. So please update the code to have ```#``` as comment character for both parsers and update the doc.